src/deepsparse/pipeline.py

```diff
@@ -760,6 +760,7 @@ def log(
             identifier=identifier,
             value=value,
             category=category,
+            pipeline_name=self._identifier(),
```

Comment:
Can this just be added as an explicit arg instead of adding kwargs? Are all loggers expected to handle this argument?

Reply:
Yeah, this is the crux. Right now, this is only done to improve UX on the `PrometheusLogger` side. If more loggers were depending on this variable, this could probably become an explicit arg; this is not the case right now tho